src/cargoaction.h

```diff
@@ -77,19 +77,17 @@ class CargoTransfer : public CargoMovement<VehicleCargoList, StationCargoList> {
 
 /** Action of loading cargo from a station onto a vehicle. */
 class CargoLoad : public CargoMovement<StationCargoList, VehicleCargoList> {
-protected:
-	TileIndex load_place; ///< TileIndex to be saved in the packets' loaded_at_xy.
 public:
-	CargoLoad(StationCargoList *source, VehicleCargoList *destination, uint max_move, TileIndex load_place) :
-			CargoMovement<StationCargoList, VehicleCargoList>(source, destination, max_move), load_place(load_place) {}
```

Comment:
Removing loaded_at_xy is better be done after #11274 as it needs load_place here and in CargoReservation.

Reply:
Yeah, let's not build these kind of constraints. Who knows when #11274 lands, while this PR is trivially correct.

Even more, it would be good for #11274 to reintroduce these variables, as `load_place` is a weird name. More commonly we use `load_xy` or `load_tile`.

Reply:
I guess #11274 can reintroduce them as `location` like in other actions as it not always load or unload.